judge/views/problem_data.py

```diff
@@ -44,7 +44,19 @@ class ProblemDataForm(ModelForm):
     def clean_zipfile(self):
         if hasattr(self, 'zip_valid') and not self.zip_valid:
             raise ValidationError(_('Your zip file is invalid!'))
-        return self.cleaned_data['zipfile']
+
+        zipfile = self.cleaned_data['zipfile']
+        if zipfile and zipfile.name == 'init.yml':
+            raise ValidationError(_('Zip files must not be named init.yml'))
```

Comment:
```suggestion
            raise ValidationError(_('Zip files must not be named init.yml.'))
```
For consistency with the other error messages.